packages/dd-trace/test/appsec/index.fastify.plugin.spec.js

```diff
@@ -262,6 +262,174 @@ withVersions('fastify', 'fastify', version => {
       }
     })
   })
+
+  describe('Suspicious request blocking - path parameters', () => {
+    let server, paramHookSpy, axios
+
+    before(() => {
+      return agent.load(['fastify', 'http'], { client: false })
+    })
+
+    before((done) => {
+      const fastify = require(`../../../../versions/fastify@${version}`).get()
+
+      const app = fastify()
+      app.get('/multiple-path-params/:parameter1/:parameter2', (request, reply) => {
+        reply.send('DONE')
+      })
+
+      app.register(async function (nested) {
+        nested.get('/:nestedDuplicatedParameter', async (request, reply) => {
```

Comment:
The reason why we have this `nestedDuplicatedParameter` in express test, is that we have two parameters with the same name in different routes.
Here, we don't have any other parameter called `nestedDuplicatedParameter`, so the property name is wrong, and should be something like `secondParam`, or in the prefix, it should be also `nestedDuplicatedParameter`.
Maybe this example does make no sense in fastify.